show-coordinates-in-multiple-formats/src/main/java/com/esri/arcgismaps/sample/showcoordinatesinmultipleformats/MainActivity.kt

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2023 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.showcoordinatesinmultipleformats
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.core.view.WindowCompat
+import com.arcgismaps.ApiKey
+import com.arcgismaps.ArcGISEnvironment
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
+import com.esri.arcgismaps.sample.showcoordinatesinmultipleformats.screens.MainScreen
+
+class MainActivity : ComponentActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        // authentication with an API key or named user is
+        // required to access basemaps and other location services
+        ArcGISEnvironment.apiKey = ApiKey.create(BuildConfig.API_KEY)
+        // remove focus from text fields when keyboard closes
+        WindowCompat.setDecorFitsSystemWindows(window, false)
+        // set compose content
+        setContent {
+            SampleAppTheme {
+                SampleApp()
+            }
+        }
+    }
+
+    @Composable
+    private fun SampleApp() {
```

Comment:
I plan to update the NewModuleScript with few learnings from writing these initial compose samples. I think this would be a good enhancement to have, would you suggest the function to be renamed to use just the sample title or something like this?
- `ShowCoordinatesInMultipleFormats()`
- `ShowCoordinatesInMultipleFormatsApp()`
- `ShowCoordinatesInMultipleFormatsSample()`

Reply:
I think the default when you create a compose app in Android Studio is to use `App` so I would suggest `ShowCoordinatesInMultipleFormatsApp()`